<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #404258;
    color: #fff;
 }
QLabel {
    border: 1px solid;
    border-color: #50577A;
    background-color: #6B728E;
}
QTextEdit {
    border: 1px solid;
    border-color: #50577A;
    background-color: #6B728E;
}
QTextEdit:focus {
    background: #7B8FA1;
    border: 2px solid;
    border-color: #50577A;
}
QLineEdit {
    border: 1px solid;
    border-color: #50577A;
    background-color: #6B728E;
}
QLineEdit:focus {
    background: #7B8FA1;
    border: 2px solid;
    border-color: #50577A;
}
 QPushButton {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #404258, stop: 1 #6B728E);
    border-radius: 6px;
    padding: 5px;
    color: white;
    font-weight: bold;
}
QPushButton:hover {;
    border: 1px solid;
    border-color: #fff;
}
QPushButton:pressed {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6B728E, stop: 1 #404258);
}</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>8</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Full name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="full_name_line_edit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Occupation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="occupation_line_edit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="submit_button">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
